src/foundry/common/config.mjs/releaseData.d.ts

```diff
@@ -0,0 +1,98 @@
+import { PropertiesToSource } from '../../../types/helperTypes';
+import { DocumentData } from '../abstract/module.mjs';
+import * as fields from '../data/fields.mjs';
+
+export interface ReleaseDataSchema extends DocumentSchema {
+  generation: typeof fields.REQUIRED_NUMBER;
+  channel: typeof fields.REQUIRED_STRING;
+  suffix: typeof fields.STRING_FIELD;
+  build: typeof fields.REQUIRED_NUMBER;
+  time: typeof fields.TIMESTAMP_FIELD;
+  node_version: typeof fields.POSITIVE_INTEGER_FIELD;
+  notes: typeof fields.BLANK_STRING;
+  download: typeof fields.BLANK_STRING;
+}
+
+export interface ReleaseDataProperties {
+  /** The major generation of the Release */
+  generation: number;
+
+  /** The channel the Release belongs to, such as "stable" */
+  channel: string;
```

Comment:
Would it make sense to type this more strictly since the set of possible channels is pretty static. Maybe a union of the known channels and string would be ideal?

Reply:
There is no enforcement of that in the field type though. So in theory another module could write whatever string in there.